spec/services/spending_breakdown/export_spec.rb

```diff
@@ -1,32 +1,26 @@
 RSpec.describe SpendingBreakdown::Export do
-  let!(:organisation) { create(:delivery_partner_organisation, beis_organisation_reference: "BC") }
-  let!(:activity) { create(:project_activity, organisation: organisation) }
-  let!(:source_fund) { Fund.new(activity.source_fund_code) }
-  let!(:actual) { create(:actual, parent_activity: activity, value: 100, financial_quarter: 1, financial_year: 2020) }
-  let!(:refund) { create(:refund, parent_activity: activity, value: -200, financial_quarter: 1, financial_year: 2020) }
+  before(:all) do
```

Comment:
This is a VERY cool approach!

Reply:
It felt a bit old-skool, but then I realised that doesn't matter - it's just much, much better and faster in this use case! 🏎️